specification/storage/resource-manager/Microsoft.Storage/stable/2021-08-01/storage.json

```diff
@@ -2919,6 +2940,16 @@
           "x-ms-client-name": "EnableHttpsTrafficOnly",
           "description": "Allows https traffic only to storage service if sets to true. The default value is true since API version 2019-04-01."
         },
+        "isSftpEnabled": {
+          "type": "boolean",
+          "x-ms-client-name": "IsSftpEnabled",
+          "description": "Enables Secure File Transfer Protocol, if set to true"
+        },
+        "isLocalUserEnabled": {
+          "type": "boolean",
+          "x-ms-client-name": "IsLocalUserEnabled",
+          "description": "Enables local users feature, if set to true"
```

Comment:
What's the default value if user not set it?

Reply:
default interpretation is false if not specified